09_dt_rtc/patches/0002-09_dt_rtc-Change-default-eeprom3-to-AT24C32.patch

```diff
@@ -0,0 +1,26 @@
+From 39293c21b4e9076a699d81a89e95fcecb5b1d2ae Mon Sep 17 00:00:00 2001
+From: Vadym Kharchuk <vadym.kharchuk@globallogic.com>
+Date: Mon, 15 Mar 2021 20:58:46 +0200
+Subject: [PATCH 2/2] 09_dt_rtc: Change default eeprom3 to AT24C32
+
+Signed-off-by: Vadym Kharchuk <vadym.kharchuk@globallogic.com>
+---
+ arch/arm/boot/dts/am335x-bone-common.dtsi | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/arch/arm/boot/dts/am335x-bone-common.dtsi b/arch/arm/boot/dts/am335x-bone-common.dtsi
+index 73b514d..d7669ec 100644
+--- a/arch/arm/boot/dts/am335x-bone-common.dtsi
++++ b/arch/arm/boot/dts/am335x-bone-common.dtsi
+@@ -287,7 +287,7 @@
+ 	};
+ 
+ 	cape_eeprom3: cape_eeprom3@57 {
+-		compatible = "atmel,24c256";
++		compatible = "atmel,24c32";
```

Comment:
Do not change (common) include file.
Override this property in the board file am335x-boneblack.dts

Reply:
Thanks. Fixed